aries_cloudagent/config/tests/test_argparse.py

```diff
@@ -222,6 +222,59 @@ async def test_transport_settings_file(self):
         )
         # no asserts, just testing that the parser doesn't fail
 
+    async def test_multitenancy_settings(self):
+        """Test required argument parsing."""
+
+        parser = argparse.create_argument_parser()
+        group = argparse.MultitenantGroup()
+        group.add_arguments(parser)
+
+        with async_mock.patch.object(parser, "exit") as exit_parser:
+            parser.parse_args(["-h"])
+            exit_parser.assert_called_once()
+
+        result = parser.parse_args(
+            [
+                "--multitenant",
+                "--multitenant-admin",
```

Comment:
I think you can skip multitenant-admin setting from this test.

Reply:
Removed